Buildings/Fluid/ZoneEquipment/FanCoilUnit/FanCoilUnit.mo

```diff
@@ -233,38 +187,41 @@ model FanCoilUnit
     dp1_nominal=0,
     dp2_nominal=0,
     UA_nominal=UAHeaCoi_nominal,
-    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial)
-                                                 "Hot water heating coil"
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial) if
+    has_heatingCoilHHW                           "Hot water heating coil"
```

Comment:
"has_heatingCoilHHW" should be defined as protected and is it supposed to be "has_heaCoiHW"?

Reply:
Changing parameter to protected was causing translation errors. Will try again with latest model

Reply:
Use reference from Buildings.Controls.OBC.CDL.Continuous.PID